internal/data/metrics.sql.go

```diff
@@ -0,0 +1,227 @@
+// Code generated by sqlc. DO NOT EDIT.
```

Comment:
nitpick: general question: is it a convention to end generated files with `.gen.go`?

Reply:
don't think if its a convention, but makes it easier to understand that this file was auto generated, and don't edit it directly.